zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/controller/usermanagement/GroupQueryControllerTest.java

```diff
@@ -193,18 +226,134 @@ void shouldSortAndPaginateSearchResult() {
             """
             {
               "sort":  [{"field": "name", "order":  "asc"}],
-              "page":  {"from":  20, "limit":  10}
+              "page":  {"from":  20, "limit":  2}
             }
-             """)
+            """)
         .exchange()
         .expectStatus()
-        .isOk();
+        .isOk()
+        .expectBody()
+        .json(EXPECTED_SEARCH_RESPONSE);
```

Comment:
❓ If the limit is set to 2 wouldn't we only expect 2 of the groups in the result?

Reply:
Something isn't workin as expected, I'll need to investigate some more.

Reply:
@remcowesterhoud since we're mocking the `GroupService` the actual sort and filter parameters are not considered. Basically, with the test we confirm that the requests and responses are properly parsed and mapped.

We have separate `GroupService` tests that cover the sorting and filtering.